players-front/src/components/UserGoals.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from "react";
+import { Button, Table } from "reactstrap";
+import './styles/Header.css';
+import { token_to_json } from './helpers/token_helpers.js';
+
+// API
+//import axios from "axios";
+import axiosWithJWT from "../axiosApi";
+
+class UserGoals extends Component {
+
+  state = {
+    goals: []
+  }
+
+  componentDidMount() {
+
+    // token_to_json is a helper function
+    let token_json = token_to_json(localStorage.getItem('access_token'));
+    console.log(token_json)
+    axiosWithJWT.get('/accounts/' + token_json.user_id + '/goals').then(
```

Comment:
At some point we'll need to verify on the backend that they're requesting data for the user that they actually are